examples/crds/ciliumnodes.yaml

```diff
@@ -253,6 +253,11 @@ spec:
                     items:
                       description: AzureInterface represents an Azure Interface
                       properties:
+                        GatewayIP:
+                          description: "GatewayIP is the interface subnet's default
+                            route \n OBSOLETE: This field is obsolete, please use
+                            Gateway field instead."
```

Comment:
I'm not aware of where this docstring is used exactly, but does `Gateway` need to be capitalized here? Could it be turned into a fully-qualified name like `items.gateway` instead? (again, not sure about this)

Reply:
This is generated from docstring of `Gateway` field in `AzureInterface` struct in `pkg/azure/types/types.go` file. If you notice other fields, they too use capitalized field name in their docstring too.